doc/orchestrator.rst

```diff
@@ -71,100 +335,217 @@ This method is deemed ``locality based inference`` since data is local to each
 process and the ``Orchestrator`` is deployed locally on each compute host where
 the distributed application is running.
 
+-------
+Example
+-------
+This example provides a demonstration on automating the deployment of
+a colocated Orchestrator within an Experiment.
 
-To create a colocated model, first, create a ``Model`` instance and then call
-the ``Model.colocate_db_tcp`` or ``Model.colocate_db_uds`` function.
+The example is comprised of two script files:
 
-.. currentmodule:: smartsim.entity.model
+* The Application Script
+* The Experiment Driver Script
 
-.. automethod:: Model.colocate_db_tcp
-    :noindex:
+**The Application Script Overview:**
+The example application script is a Python file that contains
+instructions to create and connect a SmartRedis
+client to the colocated Orchestrator.
+Since a colocated Orchestrator is launched when the Model
+is started by the experiment, you may only connect
+a SmartRedis client to a colocated database from within
+the associated colocated Model script.
 
-.. automethod:: Model.colocate_db_uds
-    :noindex:
+**The Application Script Contents:**
 
-Here is an example of creating a simple model that is colocated with an
-``Orchestrator`` deployment using Unix Domain Sockets
+1. Connecting a SmartRedis client within the application to send and retrieve a tensor
+   from the colocated database.
 
-.. code-block:: python
+**The Experiment Driver Script Overview:**
+The experiment driver script launches and manages
+the example entities with the ``Experiment`` API.
+In the driver script, we use the ``Experiment``
+to create and launch a colocated ``Model`` instance
+launches a colocated Orchestrator and runs the application
```

Comment:
wording is wonky "`to create and launch a colocated ``Model`` instance launches a colocated Orchestrator"